src/server_manager/web_app/location_name.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ServerLocation} from '../model/location';
+
+// import '@formatjs/intl-displaynames/polyfill'
+// import '@formatjs/intl-listformat/polyfill'
+
+interface Localizer {
```

Comment:
This was hard to understand. Pass the language and localize parameters explicitly where you need.

Reply:
OK, removed.